scripts/signall.sh

```diff
@@ -101,6 +105,15 @@ if [ -n "$USIGNKEY" ]; then
 		signify-openbsd -S -s "$(readlink -f "$tmpdir/usign.sec")" -m "{}" \; || finish 5
 fi
 
+if [ -n "$APKSIGNKEY" ]; then
+    umask 077
+    echo "$APKSIGNKEY" > "$tmpdir/apk.pem"
+
+    umask 022
+    find "$tmpdir/tar/" -type f -name "packages.adb" -exec \
+        "${APK_BIN:-apk}" adbsign --allow-untrusted --sign-key "$(readlink -f "$tmpdir/apk.pem")" "{}" \; || finish 6
```

Comment:
For some reason this fails:

```shell
/scripts/signall.sh /master/signing/mediatek.filogic.tar.gz master

** RUNNING ON BUILDMASTER **
 in dir /master
 argv: ['/scripts/signall.sh', '/master/signing/mediatek.filogic.tar.gz', 'master']
 env: {'CONFIG_INI': '/config/config.ini'}
gpg: keybox '/master/signall.4246/gpg/pubring.kbx' created
gpg: /master/signall.4246/gpg/trustdb.gpg: trustdb created
gpg: key CD84BCED626471F1: public key "OpenWrt Build System (PGP key for unattended snapshot builds) <pgpsign-snapshots@openwrt.org>" imported
gpg: key CD84BCED626471F1: secret key imported
gpg: Total number processed: 1
gpg:               imported: 1
gpg:       secret keys read: 1
gpg:   secret keys imported: 1
find: ‘apk’: No such file or directory
find: ‘apk’: No such file or directory
```

but the `signfiles` step is marked as successful, although there is `haltOnFailure=True` defined for this step, so in this case the build should fail. Probably some issue with exit code handling/passing somewhere?